File: src/test/java/solutions/n383_ransom_note/SolutionTest.java
package solutions.n383_ransom_note;

import org.junit.Assert;
import org.junit.BeforeClass;
import org.junit.Test;

/**
 * @author : Sobornov Vladimir
 * @since : 09.11.2022
 **/
public class SolutionTest {

    private static Solution solution;

    public SolutionTest() {
    }

    @BeforeClass
    public static void beforeClass() {
        solution = new Solution();
    }

    @Test
    public void canConstructTest() {
        Assert.assertEquals(true, solution.canConstruct("aab", "baa"));
    }
}

File: src/test/kotlin/solutions/find_occurences/SolutionTest.kt
package solutions.find_occurences

import org.junit.Assert
import org.junit.Test

class SolutionTest {
    private val solution = Solution()

    @Test
    fun findOccurrencesTest() {
        Assert.assertEquals(3, solution.findOccurrences(intArrayOf(5,6,4,3,2,9,11,12,4,7,8,4), 4))
    }
}